### PR TITLE
Add Hacktoberfest 2024 event

### PR DESCRIPTION
**Pull request by sudhanshutech**
**Description**

This PR fixes #

- Previous year form is disabled
- New event added 

**Notes for Reviewers**
Left is to finalize the slide deck so that just left to add in the event

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 10c5da1fb28aec5eb840926476c5bdcf5d1b3b0c at: https://66ecf376c7eb82d85a3d9e0d--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 24f5cfa0edc4c4a75a903316ed24225cb1ca48f4 at: https://66ecf9cef51510d9dad7f986--layer5.netlify.app

File: src/collections/events/2024/hacktoberfest-2024/hacktoberfest-contributor.png
Comment (jamieplu) [inline src/collections/events/2024/hacktoberfest-2024/hacktoberfest-contributor.png:1]: New badge needed.

File: src/collections/events/2024/hacktoberfest-2024/index.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+<ul>
+    <li><strong>DATE:</strong> September 27, 2024</li>
+    <li><strong>TIME:</strong> 10:00 AM CT | 8:30 PM IST</li>
+    <li><strong>MEET:</strong> <a href="https://meet.layer5.io/community"> Zoom</a></li>
+</ul>
+
+<p> Resources:
+    <ul>
+    <li><Link to="https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+org%3Ameshery+org%3Aservice-mesh-performance+org%3Aservice-mesh-patterns+label%3A%22hacktoberfest%22+">Issues for Hacktoberfest</Link></li>
```
Comment (leecalcote) [inline src/collections/events/2024/hacktoberfest-2024/index.mdx:35]: Make sure this has layer5labs as an org, too. Update the place where you got this link, too.

**Pull request merged**